content/en/lotus/developers/local-network.md

```diff
@@ -288,9 +288,9 @@ The port and IP can be configured by using the LIBP2P variables in your [config
 
 You should now see your new node getting synchronized with the chain. You can check the progress;
 
-    ```shell
-    ./lotus sync wait
-    ```
+```shell
+./lotus sync wait
```

Comment:
thats odd, the inline editor in Github shows these identations, but then they probably don't really exist in the file? :thinking: odd :scream:

Reply:
I pushed a couple of odd and stuid changes just now as well, so might just be that. Getting late here, so my brain is getting dumb 😅

But the formatting is:
- Needs to be indented if creating a numbered list / step-by-step list.
- Else, not indented